evm/src/main/java/org/hyperledger/besu/evm/gascalculator/GasCalculator.java

```diff
@@ -635,31 +633,37 @@ default long blobGasCost(final long blobCount) {
     return 0L;
   }
 
-  /**
-   * Compute the new value for the excess blob gas, given the parent value, the parent blob gas used
-   * and the parent target blobs per block, if present. Used from Cancun onwards. Presence of
-   * parentTargetBlobsPerBlock implies EIP-7442/Prague enabled. Default to Cancun constant target
-   * gas value if parentTargetBlobsPerBlock is not present.
+  //  /**
+  //   * Compute the new value for the excess blob gas, given the parent value, the parent blob gas
+  // used
+  //   * and the parent target blobs per block, if present. Used from Cancun onwards. Presence of
+  //   * parentTargetBlobsPerBlock implies EIP-7442/Prague enabled. Default to Cancun constant
+  // target
+  //   * gas value if parentTargetBlobsPerBlock is not present.
+  //   *
+  //   * @param parentExcessBlobGas excess blob gas from the parent
+  //   * @param parentBlobGasUsed blob gas used from the parent
+  //   * @return the new excess blob gas value
+  //   */
+  //  default long computeExcessBlobGas(
```

Comment:
Assume this all gets deleted after coming out of draft

Reply:
removed